src/FileReader/PHPFileReader.php

```diff
@@ -11,6 +12,8 @@
 
     public function read($filename)
     {
+        Assertion::string($filename);
+
         $this->filename = $filename;
 
         $this->assertFileExists();
```

Comment:
@tomphp 

And maybe even `Assertion::isArray()` for the `$config` we try to read from the file?